swap_meet/decor.py

```diff
@@ -1,2 +1,14 @@
-class Decor:
-    pass
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, category = "Decor", condition = 0):
+        self.category = category
+        self.condition = condition
+
+        #super().__init__(category = "Clothing", conditon = condition)
+
+        #super().condition_description()
```

Comment:
I say we go with line 8 instead of rewriting the attributes. Also, we know that the Decor child class will always have a category value of "Decor", so no reason to give the users the freedom to change that.

```suggestion
    def __init__(self, condition = 0):
        super().__init__(category = "Decor", condition = condition)
```